### PR TITLE
Add Achievements and Quest Completes to log book

### PR DESCRIPTION
**Pull request by jaahay**
Helpful for if you idle for extended periods of time. Also, fosters a great sense of satisfaction.

### CODE REVIEW

File: src/scripts/logBook/LogBookTypes.ts [hunk 1]
```diff
@@ -24,4 +24,12 @@ const LogBookTypes: Record<string, LogBookType> = {
         display: 'primary',
         label: 'FOUND',
     },
+    ACHIEVEMENT: {
+        display: 'success',
+        label: '\'CHIEVE',
```
Comment (RedSparr0w) [inline src/scripts/logBook/LogBookTypes.ts:29]: I'm not sure about shortening this to `'CHIEVE`

Comment (jaahay) [reply src/scripts/logBook/LogBookTypes.ts:29]: `ACHIEVE` or `ACHIEVEMENT`
I wanted to keep the text width same as the other labels

File: docs/scripts/script.min.js [hunk 2]
```diff
@@ -29480,29 +29489,29 @@ SpecialEvents.events = [];
 // TODO: Fetch events from a server each 1/2/3/6/12/24 hours?
 // Create our events here for now
 // Yearly
-SpecialEvents.newEvent('Flying Pikachu', 'Encounter Flying Pikachu for a limited time roaming Kanto.',
```
Comment (jaahay) [inline docs/scripts/script.min.js:29483]: I think Atom on Windows is proactively trying to change some file endings. Also, I think git hates me

**Comment by jaahay** [comment]
Feel free to tweak any text or other UI stuff. Much appreciated

File: docs/scripts/script.min.js (hunk 2 above)
Comment (RedSparr0w) [reply docs/scripts/script.min.js:29483]: The `docs` file shouldn't be update, we only update this when we push to master for the live website.

Comment (jaahay) [reply docs/scripts/script.min.js]: Understood! Thanks for (another) assist :)

File: src/scripts/logBook/LogBookTypes.ts [hunk 3]
```diff
@@ -24,4 +24,12 @@ const LogBookTypes: Record<string, LogBookType> = {
...
         label: 'FOUND',
     },
+    ACHIEVEMENT: {
+        display: 'success',
+        label: 'ACHIEVE',
+    },
+    QUEST_COMPLETE: {
+        display: 'success',
+        label: 'QUEST',
+    },
```
Comment (RedSparr0w) [inline src/scripts/logBook/LogBookTypes.ts:34]: Maybe we could use something like `warning` and `info` here instead to make them stand out from the rest of the logs:
```suggestion
    ACHIEVEMENT: {
        display: 'warning',
        label: 'ACHIEVE',
    },
    QUEST_COMPLETE: {
        display: 'info',
        label: 'QUEST',
    },
```

Before | After
:-- | :--
image | image

Comment (jaahay) [reply src/scripts/logBook/LogBookTypes.ts:34]: Looks sleeker with the `warning` and `info` colours

**Review by RedSparr0w — APPROVED** [review]
Looks good 👍,

Thanks for the PR!

**Pull request merged**